src/sentry/api/endpoints/user_notification_fine_tuning.py

```diff
@@ -101,7 +105,9 @@ def put(self, request, user, notification_type):
 
                 # make sure user is in org
                 if org_id not in org_ids:
-                    return Response(status=status.HTTP_403_FORBIDDEN)
+                    return Response({
+                        'detail': 'User does not belong to at least one requested org (org_id: %s).' % org_id
```

Comment:
Could we show the slug? End users don't fetch organizations by autoincrement id.

Reply:
Can do, though this isn't a publicly used endpoint (at least currently) - it's only our web app or edge cases in which support has to use our super powers to change a user's settings for them via the API. Also, the org is specified in the request by id, so whoever/whatever is making the request is already thinking in id terms rather than slug terms.

We don't currently have the org itself (only the id), so it'd take a trip to the DB to get the slug. Happy to add that if you think it's necessary, though.

Reply:
If this is primarily an internal endpoint ids are fine.